examples/example_temptest.m

```diff
@@ -0,0 +1,59 @@
+clear;
+setup;
+n = neuron.Neuron();
+n.load_file('stdrun.hoc');
+n.reset_sections();
+
+n.celsius = 6.3;
+soma = n.Section(name="soma");
+soma.insert_mechanism("hh");
+soma.nseg = 11;
+
+syn1 = n.Exp2Syn(soma(0.25));
+syn.tau1 = 0.1;
+syn.tau2 = 4;
+
+ns = n.NetStim();
+ns.start = 5;
+ns.number = 1;
+ns.noise = 0;
+
+nc = n.NetCon(ns, syn1);
+nc.weight = 1;
+nc.delay = 0;
+
+t = n.Vector();
+t.record(n.ref("t"));
+v = n.Vector();
+v.record(soma(0.5).ref("v"));
+
+nc_recorder = n.NetCon(soma(0.5).ref("v"), n.null);
+t_spikes = n.Vector();
+nc_recorder.record(t_spikes);
+
+n.finitialize(-65);
+n.continuerun(10);
+
+disp(length(t_spikes));  % 1
+disp(t_spikes(1));  % 6.1000 (on first run only)
```

Comment:
Is the first run only comment still true, or did resetting code to fixed step fix that?

Reply:
Resetting fixes that.

I also tried fixing it by making a `SavedState` at the first initialization of Neuron, which is then restored every time `n = neuron.Neuron()` is run, but that did not work.